docs/guides/chip_tool_guide.md

```diff
@@ -0,0 +1,769 @@
+    protocol and results in printing the following log:
+
+        ```
+        Secure Session to Device Established
+        ```
+
+-   Providing the device with a network interface using ZCL Network
+    Commissioning cluster commands, and the network pairing credentials from
+    step 3
+-   Discovering the IPv6 address of the Matter accessory using the SRP (Service
+    Registration Protocol) for Thread devices, or the mDNS (Multicast Domain
+    Name System) protocol for Wi-Fi or Ethernet devices. It results in printing
+    log that indicates that the node address has been updated. The IPv6 address
+    of the device is cached in the controller for later usage.
+-   Closing the Bluetooth LE connection, as the commissioning process is
+    finished and the `chip-tool` is now using only the IPv6 traffic to reach the
+    device.
+
+#### Commissioning over IP
+
+The command below will discover devices and try to pair with the first one it
+discovers using the provided _setup PIN code_.
+
+```
+./BUILD_PATH/chip-tool pairing onnetwork <node_id> <pin_code>
+
+```
+
+where:
+
+-   _<node_id\>_ is the user-defined ID of the node being commissioned,
+-   _<pin_code\>_ is device specific _setup PIN code_ determined in step 4
+
+The command below will discover devices with long discriminator and try to pair
+with the first one it discovers using the provided setup code.
+
+```
+./BUILD_PATH/chip-tool pairing onnetwork-long <node_id> <pin_code> <discriminator>
+
+```
+
+where:
+
+-   _<node_id\>_ is the user-defined ID of the node being commissioned,
+-   _<pin_code\>_ and _<discriminator\>_ are device specific keys determined in
+    step 4
+
+The command below will discover devices based on the given QR code payload
+(which devices log when they start up) and try to pair with the first one it
+discovers.
+
+```
+./BUILD_PATH/chip-tool qrcode <node_id> <qrcode_payload>
+
+```
+
+where:
+
+-   _<node_id\>_ is the user-defined ID of the node being commissioned,
+-   _<qrcode_payload\>_ is the QR code payload
+
+#### Forget the currently-commissioned device
+
+```
+./BUILD_PATH/chip-tool pairing unpair <node_id>
+
+```
+
+where:
+
+-   _<node_id\>_ is the user-defined ID of the node which is going to be forgot
+    by the `chip-tool`
+
+#### Note
+
+`chip-tool` currently only supports commissioning and remembering one device at
+a time. The configuration state is cached in `/tmp/chip_tool_config.ini`;
+deleting this and other .ini files in `/tmp` can sometimes resolve issues
+related to stale configuration.
+
+### Step 6: Control application ZCL clusters.
+
+For the lighting-app example, execute the following command to toggle the LED
+state:
+
+```
+./BUILD_PATH/chip-tool onoff toggle <node_id> <endpoint_id>
+```
+
+where:
+
+-   _<node_id\>_ is the user-defined ID of the commissioned node
+-   _<endpoint_id\>_ is the ID of the endpoint with OnOff cluster implemented
+
+To change the brightness of the LED, use the following command, with the
```

Comment:
Again, is this an alternative step or something I need to do after issuing the command from line 334?

Reply:
With a procedure in steps, we need to be extra clear about the order of steps and listing potential alternatives. So this entry and entry in line 321 could be a bullet point list with two alternatives, what do you think?

Reply:
I would not object here but for the options that are also listed as bullet points (in the whole document actually). So we would need to use different style of bullets which I am not sure is possible in Markdown (the nested bullet points are not what we want). I also worry that if I use the *alternatively* work somewhere the user may think that the commands are mutually exclusive. I will try to explain it more distinctly.